controllers/likeController.js

```diff
@@ -0,0 +1,21 @@
+const likeService = require('../services/likeService');
+const { catchAsync } = require('../utils/error');
+
+const likes = catchAsync(async (req, res) => {
+  const { userId, productId } = req.body;
+
+  const likes = await likeService.likes(userId, productId);
+  return res.status(201).json({ message: 'Success Create' });
+});
+
+const likesCensel = catchAsync(async (req, res) => {
```

Comment:
함수명은 행위를 나타낼 수 있게 동사형으로 지어주면 좋습니다.
```suggestion
const deleteLike = catchAsync(async (req, res) => {
```